src/Tools/dotnet-monitor/Extensibility/ExtensionRepositoryUtilities.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.FileProviders;
+using System.IO;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Extensibility
+{
+    public static class ExtensionRepositoryUtilities
+    {
+        public static bool ExtensionDefinitionExists(IFileProvider fileSystem, string extensionPath)
```

Comment:
```suggestion
        public static bool TryGetExtensionDefinitionPath(this IFileProvider fileProvider, string extensionName, out string definitionPath)
```

1. Make this an extension method instead of yet another utility class.
2. Let's more all of the path calculations inside this method as much as possible so that callers don't have to understand the pathing structure.